docs_src/microservices/application/V2Migration.md

```diff
@@ -0,0 +1,296 @@
+# V2 Migration Guide
+
+!!! edgey "EdgeX 2.0"
```

Comment:
Are you wanting the three exclamation marks to be there? Same goes for the later exclamation marks you add...
![image](https://user-images.githubusercontent.com/25728646/125098686-19555c80-e08c-11eb-8b8f-0abaad1e2ac9.png)

Reply:
Yes, this is special for GitHub pages that we use. It renders like this.
![image](https://user-images.githubusercontent.com/44779287/125105132-fd08ee00-e092-11eb-97c2-0a51ed7f1062.png)
 
FYI, here is the docs pages that this will land on.
https://docs.edgexfoundry.org/2.0/